### DISCUSSION TITLE
Notes and Custom fields doesn't support search

### DISCUSSION THREAD

**Discussion by taowei20** [dani-garcia/vaultwarden#4829] [Q&A] [answered]
Notes and Custom fields doesn't support search

**Comment by taowei20** [comment, marked as answer]
I'm sorry, I just find use wildcards(*) can search

**Reply by BlackDex** [reply]
Also, this is a client function. This project does not maintain the clients so we wouldn't have been able to fix it if it was a bug.

**Reply by taowei20** [reply]
ok,thanks
